src/user/wikis/Editor/Editor.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+import DeleteOutlinedIcon from "@material-ui/icons/DeleteOutlined";
+import CancelButton from "@material-ui/icons/ClearOutlined";
+import SaveButton from "@material-ui/icons/SaveOutlined";
+import "react-mde/lib/styles/css/react-mde-all.css";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import * as Showdown from "showdown";
+import ReactMde from "react-mde";
+import "./Editor.scss";
+
+class Editor extends Component {
+  state = {
```

Comment:
Good practice to use constructor and super() and initialize the state inside

Reply:
Done